bin/docker-entrypoint

```diff
@@ -1,22 +1,17 @@
 #!/bin/sh -e
 
-# If running the rails server then create or migrate existing database
-if echo "${1}" | grep -q "rails$" && [ "${2}" == "server" ]; then
-  rm -f ./tmp/pids/server.pid 2>/dev/null
-  echo "Running bundle"
-  bundle
-  echo "Running db:prepare"
-  ./bin/rails db:prepare
-elif echo "${1}" | grep -q "rails$" && [ "${2}" == "test" ]; then
-  export RAILS_ENV=test
-  echo "Running bundle"
-  bundle
-  echo "Running db:prepare"
-  ./bin/rails db:prepare
-else
-  echo "Running bundle"
-  bundle
-  echo "Skipping db:prepare"
+if [ "$RAILS_ENV" != "production" ]; then
+  if echo "${1}" | grep -q "rails$" && [ "${2}" == "server" ]; then
+    rm -f ./tmp/pids/server.pid 2>/dev/null
+    echo "Running db:prepare"
+    ./bin/rails db:prepare
+  elif echo "${1}" | grep -q "rails$" && [ "${2}" == "test" ]; then
+    export RAILS_ENV=test
+    echo "Running db:prepare"
+    ./bin/rails db:prepare
+  else
+    echo "Skipping db:prepare"
+  fi
```

Comment:
On Render migrations are handled with the `render-build.sh` script and on Fly migrations are handled with the build command within `fly.toml`. Basically, no situation yet actually needs this script to handle migrations and they're being double-called.